packages/client/types/constEnums.ts

```diff
@@ -366,6 +366,7 @@ export const enum Threshold {
   MAX_POKER_DIMENSION_NAME = 50,
   MAX_REACTJIS = 12,
   MAX_POKER_STORIES = 50,
+  MAX_GITLAB_POKER_STORIES = 20,
```

Comment:
This results in ugly logic elsewhere, but I get an error if I select/deselect 50 GitLab issues: `Query too large`. This doesn't happen with the GitHub or Jira integration. The reason is the [GitLab query](https://github.com/ParabolInc/parabol/blob/5235acc69ab0094009a15832ad9f5344a11024b1/packages/client/components/GitLabScopingSearchResults.tsx) is more complex as it needs to grab projects and then issues within those projects rather than just querying the issue. 

The complex query will be fixed in this issue: https://github.com/ParabolInc/parabol/issues/6325 I've updated the AC to include removing this.

Reply:
+2 i appreciate the context!